modules/odc-viewing-application-topology.adoc

```diff
@@ -5,29 +5,24 @@
-For more information about pod status, see the link:https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/#pod-phase[Kubernetes documentation].
-
-After you create an application and an image is deployed, the status is shown as *Pending*. After the application is built, it is displayed as *Running*.
-
-.Application topology
-image::odc_application_topology.png[]
-
-The application resource name is appended with indicators for the different types of resource objects as follows:
-
-* *CJ*: `CronJob`
-* *D*: `Deployment`
-* *DC*: `DeploymentConfig`
-* *DS*: `DaemonSet`
-* *J*: `Job`
-* *P*: `Pod`
-* *SS*: `StatefulSet`
-* image:odc_serverless_app.png[title="Serverless Application"] (Knative): A serverless application
-+
-[NOTE]
-====
-Serverless applications take some time to load and display on the *Topology* view. When you create a serverless application, it first creates a service resource and then a revision. After that, it is deployed and displayed on the *Topology* view. If it is the only workload, you might be redirected to the *Add* page. After the revision is deployed, the serverless application is displayed on the *Topology* view.
-====
+You can navigate to the *Topology* view using the left navigation panel in the *Developer* perspective. After you create an application, you are directed automatically to the *Topology view* where you are directed automatically to the Topology view where you can see the status of the application pods, quickly access the application on a public URL, access the source code to modify it, and see the status of your last build. You can zoom in and out to see more details for a particular application.
+
+The *Topology* view also provides you the option to monitor your applications using the *List* view. Use the *List View* icon to see a list of all your applications and use the *Topology View* icon to switch back to the graph view.
+
+You can customize the views as required using the following:
+
+* Use the *Find by name* field to find the required components. Search results may appear outside of the visible area; click *Fit to Screen* from the lower-left toolbar to resize the *Topology* view to show all components.
+* Use the *Display Options* drop-down list to configure the *Topology* view of the various application groupings. The options are available depending on the types of components deployed in the project:
+
+** *Mode* (*Connectivity* or *Consumption*)
+*** Connectivity: to show all the connections between different nodes in the  topology.
+*** Consumption: to shows all nodes in consumption mode without any connection in the topology.
```

Comment:
Suggest:
Connectivity: Select to show all the connections between the different nodes in the topology.
Consumption: Select to show the resource consumption for all nodes in the topology. 
(please verify with the developers that they are displaying only the resource consumption)

Reply:
Done